src/alsa/ucm2/conf.d/allwinner-edp/allwinner-edp.conf

```
@@ -0,0 +1,8 @@
+Syntax 2
+
+Comment "Allwinner EDP card"
+
+SectionUseCase."HDMI" {
+	File "Hdmi.conf"
+	Comment "HDMI/Display Port Stereo"
```

```suggestion
	Comment "Embedded DisplayPort Stereo"
```
不过eDP也有声音支持？有这方面的屏幕么？

---

准确的说是 typec-dp

---

看看是否要这么改， 这个有点混